src/Hooks/UseWindowSize.js

```diff
@@ -1,4 +1,17 @@
-import { useState, useEffect } from 'react';
+import React, {
+  createContext,
+  useContext,
+  useEffect,
+  useState,
+} from 'react';
+
+import PropTypes from 'prop-types';
+import developerWarning from '../Utils/DeveloperWarning';
+
+const responsiveContext = createContext({ isConfigured: false });
+
+const WARNING_MESSAGE =
+  'Please consider wrapping your application in a `responsiveProvider` this will drastically improve performance and will be require in v3.X.X. see: https://pebble.istreamplanet.net/#/Components/ResponsiveProvider';
```

Comment:
```suggestion
  'Please consider wrapping your application in a `<ResponsiveProvider />` this will drastically improve performance and will be require in v3.X.X. see: https://pebble.istreamplanet.net/#/Components/ResponsiveProvider';
```